package.json

```diff
@@ -65,7 +65,6 @@
     "redux-mock-store": "1.5.4",
     "redux-thunk": "2.3.0",
     "regenerator-runtime": "0.13.7",
-    "tableau-react-embed": "1.1.0",
```

Comment:
I think there are some packages that are being removed from package-lock.json are not related to this like `es6-promise`.

Reply:
I think they may be related to the now removed tableau-react-embed